cmd/kyma/sync/function/function.go

```diff
@@ -0,0 +1,101 @@
+package function
+
+import (
+	"context"
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+)
+
+const (
+	defaultNamespace = "default"
+	functions = "functions"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new init command
+func NewCmd(o *Options) *cobra.Command {
+	c := command{
+		opts:    o,
+		Command: cli.Command{Options: o.Options},
+	}
+	cmd := &cobra.Command{
+		Use:   "function",
+		Short: "Saves locally resources for requested Function.",
+		Long: `Use this command to create the local workspace with the default structure of your Function's code and dependencies. Update this configuration to your references and apply it to a Kyma cluster. 
+Use the flags to specify the initial configuration for your Function or to choose the location for your project.`, //todo
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&o.Name, "name", "", `Function name.`)
+	cmd.Flags().StringVarP(&o.Namespace, "namespace","n", defaultNamespace, `Namespace from which you want to sync Function.`)
```

Comment:
Users might have a different namespace than "default" as their defult namespace in the kubeconfig. This flag should use that if empty.
See example here:
https://github.com/kyma-project/cli/blob/bad193c82b5fa538f69335789e64bb536beb2200/cmd/kyma/create/system/system.go#L74-L76